### PR TITLE
Event store doco

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

**Comment by SzymonPobiega** [comment]
@Particular/docs-maintainers I would like to get it up there in a minimal version and extend it in subsequent PRs

File: nservicebus/eventstoretransport/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The Event Store transport **does not** rely on the projections functionality for routing messages. Instead it maintains a routing topology data structure inside the database in a dedicated event stream `nsb-exchanges`. The structure is a simplified version of AMQP 0.9.1 routing topology. 
+
+partial:topology
+
+When a message is addressed to a given exchange `e` it is copied to all exchanges (similar as AMQP `fanout` exchange) and to all queues bound to that exchange. In the example above a message sent to `ExchangeB` would go to `Queue1` and a message sent to `ExchangeA` would go to both `Queue1` and `Queue2`.
+
+## Persistence
+
+TODO
```
Comment (SimonCropp) [inline nservicebus/eventstoretransport/index.md]: I like the idea of getting a minimum version in. But isn't todo too minimum ?

Comment (weralabaj) [inline nservicebus/eventstoretransport/index.md]: @SzymonPobiega I think 2 sentences should be enough here, how about 
"EventStore transport can be used in combination with any supported [NServiceBus persistence](/nservicebus/persistence/)" + saying what persistence is used for? (e.g. pub sub?)

Comment (SzymonPobiega) [inline nservicebus/eventstoretransport/index.md]: I removed the section altogether. ES persistence will have a separate doco page. Or should I merge these two? They can be used separately but I think in most cases they will be used together.

**Comment by weralabaj** [comment]
We can link between them once both are ready.

File: nservicebus/eventstoretransport/outbox_diagram_eventstore_[,2).partial.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+```
```
Comment (weralabaj) [inline nservicebus/eventstoretransport/outbox_diagram_eventstore_[,2).partial.md]: @SzymonPobiega shouldn't be name of this file `index_topology_eventstoretransport_[2,).partial.md`

Comment (SzymonPobiega) [inline nservicebus/eventstoretransport/outbox_diagram_eventstore_[,2).partial.md]: @weralabaj it should be but this one is not included in the main doco (yet) so it does not cause build errors ;-)

**Comment by weralabaj** [comment]
@SimonCropp I tried to build this branch locally to verify all rendering is ok, but every time the engine overwrites locally copied docs dir. not sure what's the issue and if it's safe to merge?

**Comment by SimonCropp** [comment]
@weralabaj can u screen share this with me when u come online

**Pull request merged**